getpaid/backends/__init__.py

```diff
@@ -2,7 +2,7 @@
 from django.core.exceptions import ImproperlyConfigured
 from django.template.base import Template
 from django.template.context import Context
-from django.utils import six
+import six
```

Comment:
This import should not be in the first import block?

Reply:
six is a third party library, same as django. Because of that I would put them in the same imports block. If the maintainer of the repo prefers separate block I can change that. Let me know.

Another option is to get the imports sorted.

The same answer applies to other similar comments.

Reply:
I'll get the imports sorted and the whole code blackened. Thanks for your work :+1: